src/crypto/index.js

```diff
@@ -501,172 +509,220 @@ Crypto.prototype.bootstrapSecretStorage = async function({
             return key;
         };
 
+    // create a new SSSS key and set it as default
+    const createSSSS = async (opts, privateKey) => {
+        opts = opts || {};
+        if (privateKey) {
+            opts.key = privateKey;
+        }
+
+        const newKeyId = await this.addSecretStorageKey(
```

Comment:
This shadows the other `newKeyId` in the outer scope, could we rename one of them?